package/MDAnalysis/analysis/contacts.py

```diff
@@ -396,11 +396,12 @@ def __init__(self, u, selection, refgroup, method="hard_cut", radius=4.5,
             dictionary of additional kwargs passed to `method`. Check
             respective functions for reasonable values.
         start : int, optional
-            First frame of trajectory to analyse, Default: 0
+            First frame of trajectory to analyse, Default: None becomes 0.
         stop : int, optional
-            Last frame of trajectory to analyse, Default: -1
+            Frame index to stop analysis. Default: None becomes
+            n_frames. Iteration stops *before* this frame number.
```

Comment:
, which means that the trajectory would be read until the end (because the first frame has index 0 and the last frame has index n_frames - 1.